src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/reference/Cas2ApplicationStatusSeeding.kt

```diff
@@ -55,18 +55,114 @@ object Cas2ApplicationStatusSeeding {
+            id = UUID.fromString("f58267b8-f91b-4a4f-9aa2-80089ba111e4"),
+            statusId = UUID.fromString("9a381bc6-22d3-41d6-804d-4e49f428c1de"),
+            name = "withdrawnByReferrer",
+            label = "Withdrawn by referrer",
+          ),
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("ed5d529c-d7d1-4f29-a0c0-89fd104cc320"),
+            statusId = UUID.fromString("9a381bc6-22d3-41d6-804d-4e49f428c1de"),
+            name = "rehousedByAnotherLandlord",
+            label = "Rehoused by another landlord",
+          ),
+        ),
       ),
       Cas2PersistedApplicationStatus(
         id = UUID.fromString("004e2419-9614-4c1e-a207-a8418009f23d"),
         name = "withdrawn",
         label = "Referral withdrawn",
         description = "The prison offender manager (POM) withdrew the application.",
+        statusDetails = listOf(
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("e4a2391e-e847-427a-a913-51e0b0ad9f52"),
+            statusId = UUID.fromString("004e2419-9614-4c1e-a207-a8418009f23d"),
+            name = "hdcNoLongerEligible",
+            label = "HDC - No longer eligible",
+          ),
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("c5dce0d2-fc05-4a07-8157-25b8821cdb06"),
+            statusId = UUID.fromString("004e2419-9614-4c1e-a207-a8418009f23d"),
+            name = "governorDecidedUnsuitable",
+            label = "Governor - Decided unsuitable",
+          ),
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("3fb37f85-be88-4eee-812d-af122e268eef"),
+            statusId = UUID.fromString("004e2419-9614-4c1e-a207-a8418009f23d"),
+            name = "governorChosenAlternative",
+            label = "Governor - Chosen alternative accommodation",
+          ),
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("8a857a7d-94f9-43ec-963c-2a2528e88a6e"),
+            statusId = UUID.fromString("004e2419-9614-4c1e-a207-a8418009f23d"),
+            name = "governorOther",
+            label = "Governor - Other",
+          ),
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("d3c789b8-947d-4e24-9cef-335545d85abe"),
+            statusId = UUID.fromString("004e2419-9614-4c1e-a207-a8418009f23d"),
+            name = "withdrewOrDeclinedOffer",
+            label = "Withdrew or declined offer",
+          ),
+        ),
       ),
       Cas2PersistedApplicationStatus(
         id = UUID.fromString("f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9"),
         name = "cancelled",
         label = "Referral cancelled",
         description = "The application has been cancelled.",
+        statusDetails = listOf(
+          Cas2PersistedApplicationStatusDetail(
+            id = UUID.fromString("f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9"),
+            statusId = UUID.fromString("fc0b7e04-7eb1-427b-b2b7-85a109a067b0"),
```

Comment:
are these statusIds meant to match the 'parent' status id? if yes, they don't appear to here? do we actually need this statusId field (I had a quick look and couldn't find any usage, but i could be wrong!)

Reply:
ah yeah that's a good point, I don't think we need the `statusId` at all here...